spec/spec_helper.rb

```diff
@@ -6,6 +6,7 @@
 require 'bundler/setup'
 Bundler.require(:default, :development)
 require 'soapy_cake'
+require 'webmock/rspec'
```

Comment:
Webmock blew up after the last update. See: https://github.com/bblimke/webmock#upgrading-from-v1x-to-v2x